src/main/resources/library/attribute-definitions.json

```diff
@@ -257,7 +265,8 @@
         {"property": "library:broadInternalResearchProjectID"},
         {"property": "library:broadInternalResearchProjectName"},
         {"property": "library:broadInternalCohortName"},
-        {"property": "library:broadInternalSeqProjectNumbers", "inputHint": "Broad internal IDs"}
+        {"property": "library:broadInternalSeqProjectNumbers", "inputHint": "Broad internal IDs"},
+        {"property": "library:lmsvn", "editable": false}
```

Comment:
Good catch ... it was originally only in one place, but with Matt's refactor, I needed to add it here and didn't remove it from above. I'm ambivalent about having these kinds of property attributes spread out at all - I'd really like these to be one thing so we don't have duplication.